NuGet.config

```diff
@@ -2,8 +2,10 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
+    <add key="AspNetCoreRelease" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />    
     <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="Rel" value="https://www.myget.org/F/signalr-alpha/api/v3/index.json" />
```

Comment:
We should only need:
<add key="Rel" value="https://www.myget.org/F/signalr-alpha/api/v3/index.json" />
<add key="NuGet" value="https://api.nuget.org/v3/index.json" />
<add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />